src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -167,6 +170,14 @@ export default {
 			}
 			this.$refs.actionsBreadcrumb.$refs.actions.opened = false
 		},
+		/**
+		 * Call the resize function after a delay
+		 */
+		delayedResize() {
+			this.$nextTick(() => {
+				setTimeout(() => this.handleWindowResize(), 110)
```

Comment:
The timeout here is necessary since the app-navigation toggle is animated with 100 ms duration.

Reply:
AH, maybe we should make sure the event is trigerred after the animation then?

Reply:
If that is possible, it would be great. No idea how, though.

Reply:
Let me fix too :p